.github/CODEOWNERS

```diff
@@ -5,6 +5,7 @@ regression/config.yaml @vectordotdev/vector @vectordotdev/single-machine-perform
 docs/ @vectordotdev/ux-team @vectordotdev/documentation
 website/ @vectordotdev/ux-team
 website/content  @vectordotdev/documentation
+website/cue/reference @vectordotdev/documentation
```

Comment:
I originally intentionally didn't tag them on all cue updates since if the cue changes don't actually appear in the docs then it doesn't seem like they need to review; however, I see in the linked PR that the VRL docs use the cue data in a way where they don't generate a static doc that could be reviewed. Maybe we could limit this to `website/cue/reference/remap`?